content/en/docs/ops/diagnostic-tools/component-logging/index.md

```diff
@@ -20,9 +20,9 @@ Logging messages output by a component are categorized by *scopes*. A scope repr
 you can control as a whole. Different components have different scopes, depending on the features the component
 provides. All components have the `default` scope, which is used for non-categorized log messages.
 
-As an example, as of this writing, `istioctl` has 24 scopes, representing different functional areas within the command:
+As an example, as of this writing, `istioctl` has 26 scopes, representing different functional areas within the command:
 
-- `ads`, `adsc`, `analysis`, `attributes`, `authn`, `authorization`, `cache`, `cli`, `default`, `grpcAdapter`, `installer`, `mcp`, `model`, `patch`, `processing`, `resource`, `secretfetcher`, `source`, `spiffe`, `tpath`, `translator`, `util`, `validation`, `validationController`
+- `ads`, `adsc`, `analysis`, `authn`, `authorization`, `ca`, `cache`, `cli`, `default`, `klog`, `kube`, `mcp`, `model`, `pkica`, `processing`, `resource`, `rootcertrotator`, `secretfetcher`, `serverca`, `source`, `spiffe`, `status`, `validation`, `validationController`, `validationServer`, `wle`
 
```

Comment:
From the 1.9.0-beta.0 `istioctl options help`:
```
Comma-separated minimum per-scope logging level of messages to output, in the form of <scope>:<level>,<scope>:<level>,... where scope can be one of [ads, adsc, all, analysis, authn, authorization, ca, cache, cli, default, installer, klog, mcp, model, patch, processing, resource, source, spiffe, tpath, translator, util, validation, validationController, wle]
```
doesn't match the list you have

Reply:
Thanks @ericvn 

This list now matches the output from `istioctl options help`.